internal/telemetrygen/common/config.go

```diff
@@ -0,0 +1,165 @@
+// Licensed to The OpenTelemetry Authors under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. The OpenTelemetry Authors licenses this file to you under
```

Comment:
I'm using "The OpenTelemetry Authors", but the file is not identical to the one in upstream. I've also made some changes since then, so my instinct is that it should be us instead of The OpenTelemetry Authors

Reply:
updated license headers